code/controllers/subsystem/SStimer.dm

```diff
@@ -550,6 +544,20 @@ SUBSYSTEM_DEF(timer)
 	prev = null
 	bucket_list[bucket_pos] = src
 
+/**
+ * Returns debug information about timer
+ */
+/datum/timedevent/proc/getTimerInfo()
+	var/static/list/bitfield_flags = list("TIMER_UNIQUE", "TIMER_OVERRIDE", "TIMER_CLIENT_TIME", "TIMER_STOPPABLE", "TIMER_NO_HASH_WAIT", "TIMER_LOOP")
+	name = "Timer: [id] (\ref[src]), TTR: [timeToRun], wait:[wait] Flags: [jointext(bitfield2list(flags, bitfield_flags), ", ")], \
```

Comment:
Im not sure, but do we want to cache this in the `name` var? I'm not sure if any of these variables change
doing so would just be like
```
if(!name)
    name = "Timer: ...
```

Reply:
Yes, that seems much better